src/ProjectOrigin.RequestProcessor/README.md

```diff
@@ -0,0 +1,45 @@
+# Layer 4 - Privacy
+
+![C4 component diagram of layer 4](/doc/layer4_privacy/component_diagra.drawio.svg)
+
+## Requests
+
+### IssueProduction
+Role: IssuingBody
+RequestBody: {m, r, from, to, gridArea, productionType..., publicM: bool }
+
+### IssueConsumption
+Role: IssuingBody
+RequestBody: {m, r, from, to, gridArea, }
+
+### TransferOwnership
+Role: Owner
+RequestBody: {m, r_transfer, r_remainer}
+
+
+```protobuf
+message IssueProduction {
+  required int64 m = 1; // quantity to transfer
+  required bytes r_transfer = 2; // r to hide transfer quantity behind
+  required bytes r_remainer = 3; // r to hide remainder behind
+}
+```
+
+
+### AllocateProductionClaim
+Role: Owner
+Description: Allocate part of a production certificate for a claim flow.
+RequestBody: {m, r_claim, r_remainer, consumptionEventId, productionEventId, ref1, ref2}
+
+### ClaimToConsumption
+Role: Owner
+Description: Initiate claim to a consumption certificate as part of claim flow.
+RequestBody: {m, r_claim, r_remainer, consumptionEventId, productionEventId, ref1, ref2}
+
+### ProductionClaimCommit
```

Comment:
Role Registry or registry operator should be updated and included in the RFC when we have validated, demonstrated and presented the walking skeleton for the ETT working group

Reply:
Hmm, don't event think the readme is up to date anymore, will look at it tomorrow.